tracee/argprinters.go

```diff
@@ -219,7 +220,7 @@ func PrintSocketType(st uint32) string {
 		10: "SOCK_PACKET",
 	}
 	var f []string
-	if stName, ok := socketTypes[st]; ok {
+	if stName, ok := socketTypes[st&0xf]; ok {
```

Comment:
Why is this needed?

Reply:
seems that only the last byte is used to encode the type: https://elixir.bootlin.com/linux/v5.5.3/source/arch/mips/include/asm/socket.h#L42
(also, look at tracee python)